exp/hubble/cmd/main.go

```diff
@@ -0,0 +1,17 @@
+// +build go1.13
```

Comment:
You need a space between the comment that has build constraints and the package name, otherwise the comment is a package comment and won't be interpreted as a build tag constraint.

That's why the builds are failing with:
```
exp/hubble/pipeline.go:1:1: +build comment must appear before package clause and be followed by a blank line
```

```suggestion
// +build go1.13

```